openrouteservice/src/main/java/org/heigit/ors/routing/graphhopper/extensions/flagencoders/VehicleFlagEncoder.java

```diff
@@ -49,7 +49,7 @@
 
     private double accelerationModifier = 0.0;
 
-    protected boolean speedTwoDirections = false;
+    protected boolean speedTwoDirections = true;
 
```

Comment:
Why has been the default changed from `false` to `true`? GH flag encoders seem to still fall back to `false`.

Reply:
this I just forgot to set back after I realised that didn't fix the HeavyVehicleFlagEncoder.